requests_kerberos/kerberos_.py

```diff
@@ -96,11 +99,17 @@ def generate_request_header(self, response):
         """
         host = urlparse(response.url).hostname
 
+        # Flags used by kerberos module.
+        gssflags = kerberos.GSS_C_MUTUAL_FLAG | kerberos.GSS_C_SEQUENCE_FLAG
+        if self.delegate:
+            gssflags |= kerberos.GSS_C_DELEG_FLAG
+
         try:
             result, self.context[host] = kerberos.authGSSClientInit(
-                "{0}@{1}".format(self.service, host))
+                "{0}@{1}".format(self.service, host), gssflags=gssflags)
         except kerberos.GSSError:
```

Comment:
Ahh, yes. I was reading one except block too far down. Considering we log the exception here, it should probably be logged in the other blocks too.